.github/workflows/fendermint-publish.yaml

```diff
@@ -74,7 +74,7 @@ jobs:
           VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
 
           # This strips the "v" prefix from the tag name.
-          [[ "${{ github.ref }}" == "refs/tags/"* ]] && VERSION=$(echo $VERSION | sed -e 's/^v//')
+          # [[ "${{ github.ref }}" == "refs/tags/"* ]] && VERSION=$(echo $VERSION | sed -e 's/^v//')
```

Comment:
Should this line be removed then?

Reply:
This whole thing was from an example from Github docs, so I kinda left it as reference so we don't have to remember how to massage tags. Could be removed and we're happy with what it does.